pkg/client/unversioned/daemon_sets_test.go

```diff
@@ -14,7 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package unversioned
+package unversioned_test
```

Comment:
That's the blackbox testing feature. https://splice.com/blog/lesser-known-features-go-test/

And for the package name, I think the convention in this project is using under score.